src/LightGraphs.jl

```diff
@@ -3,7 +3,7 @@ module LightGraphs
 
 using Requires
 using GZip
-using StatsBase: rand_binom
```

Comment:
I'm not too sure about this, but it looks like `StatsBase` has replaced `rand_binom` with  `binomrand`. Can someone take a look at this?